ecommerce/ecommerce/settings.py

```diff
@@ -11,18 +11,30 @@
 """
 
 from pathlib import Path
+import environ
+import os
 
-# Build paths inside the project like this: BASE_DIR / 'subdir'.
-BASE_DIR = Path(__file__).resolve().parent.parent
+env = environ.Env(
+    # set casting, default value
+    DEBUG=(bool, False)
```

Comment:
are we sure if debug value will be False by default?

Reply:
Thanks for the review. I guess setting False by default is a proper way. If the DEBUG is not set in the .env file, the default value will be assumed. However, if not set by default it will throw an exception as in the image below. 

![image](https://user-images.githubusercontent.com/60389950/143659111-c6fa7de3-4036-4198-b498-294d48454460.png)

If you wonder why setting False instead of True, I guess it should be more accurate that the variables of the development stage are not hardcoded in the code, although not quite sure.

![image](https://user-images.githubusercontent.com/60389950/143659370-3cd9670c-72e8-473e-972c-22ace0aab7a5.png)